lib/mayaUsd/ufe/UsdUndoAttributesCommands.h

```diff
@@ -96,14 +96,14 @@ class UsdRemoveAttributeCommand : public UsdUndoableCommand<Ufe::UndoableCommand
 }; // UsdRemoveAttributeCommand
 
 //! \brief Implementation of RenameAttributeCommand
-class UsdRenameAttributeCommand : public UsdUndoableCommand<Ufe::UndoableCommand>
+class UsdRenameAttributeCommand : public UsdUndoableCommand<Ufe::AttributeValuedUndoableCommand>
```

Comment:
Is AttributeValuedUndoableCommand in a checked-in version of UFE?  I can't find it in UFE master.

Reply:
Yes, it is here: https://git.autodesk.com/media-and-entertainment/ufe/pull/308

Reply:
Update, following [this](https://git.autodesk.com/media-and-entertainment/ufe/pull/308#discussion_r3030432) discussion for UFE, I've updated the names.